src/HistoryAPI.ts

```diff
@@ -364,14 +412,16 @@ type FromToContextRequest = Request<
     from: string
     to: string
     context: string
+    format: string
   }
 >
 
 const getFromToContext = ({ query }: FromToContextRequest, selfId: string) => {
```

Comment:
the function needs renaming, `getCommonParams` for example, because now it returns more than from to and context